### DISCUSSION TITLE
Github Actions error reporting as Annotation

### DISCUSSION THREAD

**Discussion by StefanGies99** [community/community#25176] [Actions]
<p>Hi folks,</p>
<p>I’m currently formatting failures of RSpec to set error messages as described on the <a href="https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message" rel="nofollow noopener">reference</a> page.</p>
<p>Currently, there is only the possibility to do this in one line because when I use</p>
<pre><code>/n
</code></pre>
<p>in my code it’s not picked up by Github Actions.</p>
<p>Is there a way to do this multiline instead of making use of the Github Checks/Annotations API?</p>
<p>Otherwise, it would be a nice feature in the future to make this possible.</p>
<p>Cheers,</p>
<p>Stefan Gies</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/gieforce">@gieforce</a> ,</p>
<p>You can replace ‘/n’ with ‘%0A’ for mutliple strings output. Code sample as below:</p>
<pre><code>steps:
      - name: error log
        run: |
          text="This is error1\nThis is error2"
          text="${text//'\n'/'%0A'}" #replace '\n' with '%0A'
          echo $text
          echo "::error file=app.js,line=10,col=15::$text"
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/15c264f4e4635cf9478e5556ef14d6fd724f37bd.png" data-download-href="/uploads/short-url/36uvVc0hlSTIW4payGG0f7jWY2h.png?dl=1" title="multipleline.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181089247-7c24692a-ab01-4216-bfa1-f7a19e17ac16.png" alt="multipleline.png" data-base62-sha1="36uvVc0hlSTIW4payGG0f7jWY2h" width="690" height="197" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/15c264f4e4635cf9478e5556ef14d6fd724f37bd_2_690x197.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/15c264f4e4635cf9478e5556ef14d6fd724f37bd_2_1035x295.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/15c264f4e4635cf9478e5556ef14d6fd724f37bd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/15c264f4e4635cf9478e5556ef14d6fd724f37bd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">multipleline.png</span><span class="informations">1324×379 14.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hope it helps!</p>
